### PR TITLE
Changelog for v2.152.0

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,99 @@
...
+
+  ``taxonomy``
+   Add a ``doc`` value to the interface.
+
+  **Updated Types**
+
+  ``time``
+    The ``time`` type now recognizes RFC822 formatted time strings.
+
+  ``biz:servicee:type:taxonomy``
```
Comment (MichaelSquires) [inline CHANGELOG.rst:38]:
```suggestion
  ``biz:service:type:taxonomy``
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -6,6 +6,99 @@
...
+
+  ``it:dev:repo:label``
+    A developer selected label.
+
+  ``it:dev:repo:issue:label``
+    A label applied to a repository issue
+
+Features and Enhancements
+-------------------------
+
```
Comment (MichaelSquires) [inline CHANGELOG.rst:75]: Remove empty line?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -6,6 +6,99 @@
...
+  ``$lib.null`` and ``$lib.undef``. Previously these printed ``None`` and an
+  opaque Python object repr.
+  (`#3361 <https://github.com/vertexproject/synapse/pull/3361>`_)
+- The ``synapse.tools.aha.list`` CLI tool now checks if it is connected to an
+  Aha service prior to enumerating Aha services.
+  (`#3371 <https://github.com/vertexproject/synapse/pull/3371>`_)
+
+Bugfixes
+--------
+- Update the ``file:path`` support for scrape related APIS to address an
```
Comment (MichaelSquires) [inline CHANGELOG.rst:86]:
```suggestion
- Update the ``file:path`` support for scrape related APIs to address an
```

**Review by MichaelSquires — CHANGES_REQUESTED** [review]
Looks great! Just a couple of minor corrections.

File: CHANGELOG.rst [hunk 4]
```diff
@@ -6,6 +6,99 @@
...
+  ``risk:threat:type:taxonomy``
+    The ``taxonomy`` interface has been added to the type.
+
+  **New Forms**
+
+  ``it:dev:repo:label``
+    A developer selected label.
+
+  ``it:dev:repo:issue:label``
+    A label applied to a repository issue
```
Comment (Cisphyx) [inline CHANGELOG.rst:71]: missing period

File: CHANGELOG.rst [hunk 5]
```diff
@@ -6,6 +6,99 @@
...
+
+Features and Enhancements
+-------------------------
+
+- Update the Storm string repr for ``$lib.null`` and ``$lib.undef`` values to
+  ``$lib.null`` and ``$lib.undef``. Previously these printed ``None`` and an
+  opaque Python object repr.
+  (`#3361 <https://github.com/vertexproject/synapse/pull/3361>`_)
+- The ``synapse.tools.aha.list`` CLI tool now checks if it is connected to an
+  Aha service prior to enumerating Aha services.
```
Comment (Cisphyx) [inline CHANGELOG.rst:81]: should this be "Aha **server** prior to enumerating Aha services"?

**Review by mikemoritz — APPROVED** [review]

**Pull request merged**